runtime/litentry/src/transaction_payment.rs

```diff
@@ -31,16 +31,12 @@ where
 	<R as frame_system::Config>::Event: From<pallet_balances::Event<R>>,
 {
 	fn on_nonzero_unbalanced(amount: NegativeImbalance<R>) {
-		let numeric_amount = amount.peek();
-		let author = <pallet_authorship::Pallet<R>>::author();
+		let _numeric_amount = amount.peek();
+		let _author = <pallet_authorship::Pallet<R>>::author();
 		<pallet_balances::Pallet<R>>::resolve_creating(
 			&<pallet_authorship::Pallet<R>>::author(),
```

Comment:
already get the value as _author, why not just use it.

Reply:
emmmmm. Yes